back-end/src/test/resources/application-test.yml

```
@@ -14,3 +14,6 @@ spring:
 server:
   http.port: 80
   port: 443
+
+cors:
+  allowed-origins: "http:localhost"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)